src/routes/safe/components/Transactions/TxList/index.tsx

```diff
@@ -7,14 +7,18 @@ import { SAFE_NAVIGATION_EVENT, useAnalytics } from 'src/utils/googleAnalytics'
 import { HistoryTransactions } from './HistoryTransactions'
 import { QueueTransactions } from './QueueTransactions'
 import { ContentWrapper, Wrapper } from './styled'
+import { createBrowserHistory } from 'history'
+import { TRANSACTIONS_PAGE_TABS } from './utils'
 
 const items: Item[] = [
```

Comment:
Do you need this when you've already added `enum TRANSACTIONS_PAGE_TABS`?

Reply:
We are using the `Tab` from safe-react-components where the `items` prop is of type `Item[]`. How would you do it?

Reply:
In that case, you needn't worry. Do you then need the `enum`? Or could you substitute it for this instead?